ch04.asciidoc

```diff
@@ -236,7 +236,7 @@ To visualize multiplication of a point with an integer, we will use the simpler
 
 [TIP]
 ====
-((("OpenSSL cryptographic library")))Most bitcoin implementations use the http://bit.ly/1ql7bn8[OpenSSL cryptographic library] to do the elliptic curve math. For example, to derive the public key, the function +EC_POINT_mul()+ is used.((("", startref="KAover04")))
+((("secp256k1 optimized C library")))Bitcoin uses the https://github.com/bitcoin-core/secp256k1[secp256k1 optimized C library] to do the elliptic curve math. For example, to derive the public key, the function +EC_POINT_mul()+ is used.((("", startref="KAover04")))
```

Comment:
secp256k1 does not have an `EC_POINT_mul` function. A similar analog might be `secp256k1_ec_pubkey_create` as used in src/key.cpp .

Reply:
@TheCharlatan 

Thank you; 3234b12.

Pull request welcome if you'd like to help expand this tip.